Resources/Maps/northway.yml

```
@@ -3288,6 +3288,13 @@
       rot: 1.5707963267948966 rad
       pos: -26.5,-1.5
       parent: 2
+- proto: APCElectronics
+  entities:
+  - uid: 3506
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)